src/server/helpers/errorHandler.ts

```diff
@@ -0,0 +1,15 @@
+import { BadRequestError, CredentialError, InternalServerError } from './errors'
+
+const Identification_Duplicated = 'IdentificationDuplicated'
+const Credential_Error = 'CredentialError'
```

Comment:
The best practice for something like this is using UPPERCASE. `IDENTIFICATION_DUPLICATED = ........`

Same with, for example, Redux action. `const ADD_USER = 'ADD_USER'`

Reply:
I'll fix it 👍🏼